services/service-tester.js

```diff
@@ -87,14 +93,16 @@ class ServiceTester {
   /**
    * Register the tests with Mocha.
    */
-  toss() {
+  toss(options) {
     const specs = this.specs
 
     const fn = this._only ? describe.only : describe
     // eslint-disable-next-line mocha/prefer-arrow-callback
     fn(this.title, function() {
       specs.forEach(spec => {
-        spec.toss()
+        if (!options.skipIntercepted || !spec.intercepted) {
```

Comment:
It's a bit difficult to follow how `skipIntercepted` is getting passed around here. I'm not sure I have a better solution, though.

Reply:
I used an environment variable to set this option via test-config and code looks better now in opinion. It's a separate commit and revert it if required https://github.com/badges/shields/pull/2195/commits/0974be49dfc5a3ae07053983849404f9d3febe4e